src/test/groovy/org/gradle/profiler/AndroidStudioIntegrationTest.groovy

```diff
@@ -314,4 +325,14 @@ class AndroidStudioIntegrationTest extends AbstractProfilerIntegrationTest {
         ]
         new Main().run(*args)
     }
+
+    private static Optional<String> findAndroidSdkPath() {
+        if (System.getenv("ANDROID_SDK_ROOT") != null) {
+            return Optional.of(System.getenv("ANDROID_SDK_ROOT").replace("\\", "/"))
+        }
+        String userDirAndroidSdkPath = "${System.getProperty("user.home").replace("\\", "/")}/Library/Android/sdk"
```

Comment:
🤔 It would be good to have a named method to call that converts Windows to Unix paths. There might already be one on the classpath we can call instead of spelling out this replace multiple times.

Reply:
Indeed there is on the classpath. Found `org.apache.commons.io.FilenameUtils.separatorsToUnix`